src/components/text-editor/TextEditor.tsx

```diff
@@ -0,0 +1,158 @@
+// Custon ESlint rule overide fixes a conflict between ESlint and Prettier
+// rules on a certain weird indentation edge case on line 74-76. on var input
+/* eslint @typescript-eslint/indent: 0 */
+import React, { useState } from "react";
+import styled, { AnyStyledComponent } from "styled-components";
+
+const Main: AnyStyledComponent = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  height: 100%;
+  width: 100%;
+  text-align: center;
+`;
+
+const Terminal: AnyStyledComponent = styled.form`
```

Comment:
does there need to be a form around the text element, since you are controlling it with react it shouldn't be strictly necessary